src/cli.lisp

```diff
@@ -20,12 +20,12 @@
      :description "Print this help menu."
      :short #\h
      :long "help")
-    (:name :upgrade
-     :description "Upgrade Qi (pull the latest from git)"
+    (:name :update
```

Comment:
What do you think about keeping `upgrade` and adding `update` instead of replacing it? An easy way to upgrade the qi code itself is nice, but I wonder if it would be confusing.

Upgrading code from master isn't _great_, but until there are releases I suppose it's not too bad.